### DISCUSSION TITLE
Goto specific TOC items from command-line

### DISCUSSION THREAD

**Discussion by qingshuizheng** [ahrm/sioyek#604] [Q&A] [answered]
Hi @ahrm , 

I am trying to figure out how to go to a specific TOC (table of content) item from command line, but got no clue how to achieve it.

Let's say that there are TOC item(s) named "references", and I wanna jump to it from outside sioyek. If "references" is unique, then go to it; otherwise, just show all the "references" items in the TOC search frame.

What I've tried: 
```
sioyek --execute-command goto_toc --execute-command-data "references"
```
The result:
Only command `goto_toc` is executed and the TOC search frame is presented, but without text "references" being inserted.

Environment:
sioyek version: https://github.com/hexomancer/sioyek/tree/v0.31.425
macOS Big Sur 11.6

Really appreciate for any help.  :-)
Zheng

**Comment by ahrm** [comment, marked as answer]
It is not currently possible because `goto_toc` is not really a command with text data. It is a command that shows the menu and jumps to the selected item.

**Reply by qingshuizheng** [reply]
Ahh that's a pity, but still thanks.
